index.html

```diff
@@ -1,14 +1,254 @@
+              opening.</p>
+          </li>
+          <li class="programLi" tabindex="0">
+            <img class="programImg" src="style/program/exhibition.svg" alt="exhibition icon">
+            <h2 class="programLiH2">exhibition</h2>
+            <p class="programLiDesc">Appreciate various creations applying architecture and design of each artist.</p>
+          </li>
+          <li class="programLi" tabindex="0">
+            <img class="programImg" src="style/program/forum.svg" alt="forum icon">
+            <h2 class="programLiH2">forum</h2>
+            <p class="programLiDesc">Have the time to share your thoughts and opinions with experts for each topic.
+            </p>
+          </li>
+          <li class="programLi" tabindex="0">
+            <img class="programImg" src="style/program/workshop.svg" alt="workshop icon">
+            <h2 class="programLiH2">workshop</h2>
+            <p class="programLiDesc">Try creating your own work using open source license rather than just watching at
+              it.
+            </p>
+          </li>
+          <li class="programLi" tabindex="0">
+            <img class="programImg" src="style/program/ignite.svg" alt="ignite icon">
+            <h2 class="programLiH2">ignite</h2>
+            <p class="programLiDesc">get opportunities to network with CC affiliates around the world, also after the
+              summit.</p>
+          </li>
+        </ul>
+        <button type="button" aria-label="Join the festival" id="programButton"><a id="programLink" href="#"
+            target="_blank" rel="noopener noreferrer">Join A&D Festival 2023</a></button>
+      </div>
+
+    </section>
+    <!-- FEATURED SPEAKERS -->
+    <section id="speakersCont">
+
+      <h2 id="speakersTitle">Featured Speakers</h2>
+
+      <div id="speakersGrid">
+
+      </div>
+
+      <script src="/assets/dynamicSpeakers.js"></script>
+      <button type="button" class="more">more<img class="arrow" src="style/icons/arrow-down.svg"
+          alt="arrow down"></button>
+
+    </section>
+
+  </section>
+  <!--   HOMEPAGE -->
+
+  <!-- ABOUTPAGE -->
+  <section id="aboutPage" class="switch">
+
+    <section id="heroContAbout">
+
+      <div id="heroAbout">
+        <h3 class="heroTItleAbout">"hello designers of the world!"</h3>
+        <h1 class="heroTItleAbout">architecture and designers festival <br>2023</h1>
+        <p id="aboutDescription">The Architects and Designers Festival brings together a community of experts,
+          academics, and activists in the field of architecture and design in a different location every two years.
+          This year, we aim to expand our guest list to include organizations and individuals interested in
+          collaborating with us on joint projects that promote the advancement of the industry, free expression, and
+          open knowledge. If you're actively involved in the world of architecture and design - advocates for
```

Comment:
- As requested by the previous reviewer kindly make sure you have two HTML files (e.g: index.html and about.html) as per the project requirements :recycle:

Reply:
@GedeonTS Oh my god sorry, I didn't commit the last changes but I already did it, it is solved! here it's the 3:36AM I'm a little tired